domain/src/main/java/me/androidbox/domain/alarm_manager/AlarmItem.kt

```diff
@@ -15,7 +15,7 @@ fun Event.toAlarmItem(agendaType: AgendaType): AlarmItem {
         agendaId = this.id,
         title = this.title,
         description = this.description,
-        remindAt = this.remindAt * 1000,
+        remindAtInMilliSecs = this.remindAt * 1000,
```

Comment:
@philipplackner Is it better to change the name here and to something more readable. As the 
```
alarmManager.setExact(
                AlarmManager.RTC_WAKEUP,
                alarmItem.remindAtInMilliSecs
               ......)
```

Needs it in milli seconds

Reply:
What I prefer is to have remindAt be the zoneddatetime, and then on the alarm manager convert it to milliseconds

Mostly so you always know you'll receive a valid date on the AlarmItem

Reply:
@philipplackner what do you think.